travel_platform/izi_travel_client.py

```diff
@@ -0,0 +1,306 @@
+        response = requests.get(url, params, headers=self.headers)
+        self._check_response_status(response)
+        tourist_attractions = []
+        if response.json():
+            tour_data = response.json()[0]["content"][0]
+            for child in tour_data["children"]:
+                attraction = {}
+                if child.get("type") == "tourist_attraction":
+                    attraction["title"] = child["title"]
+                    attraction["description"] = child["desc"]
+                    attraction["location"] = child["location"]
+                    attraction["language"] = child["language"]
+                    attraction["images"] = child["images"]
+                    attraction["attr_uuid"] = child["uuid"]
+                    tourist_attractions.append(attraction)
+            return tourist_attractions
+        return tourist_attractions
+
+    def get_object_reviews_and_rating(self, obj_uuid):
+        """
+        General method for getting reviews on any object by uuid.
+        Args:
+            obj_uuid (str): Object uuid.
+        Returns:
+            dict: Reviews info.
+        """
+        url = "{base_url}mtgobjects/{obj}/reviews".format(
+            base_url=self.api_url, obj=obj_uuid)
+        params = self._prepare_params()
+        response = requests.get(url, params, headers=self.headers)
+        self._check_response_status(response)
+        if response.json():
+            reviews = response.json().get("metadata")
+            return reviews
+        reviews = {}
+        return reviews
+
+    def get_featured_content(self):
+        """
+        Returns:
+            Website featured content.
+        """
+        url = "{}featured".format(self.api_url)
+        params = self._prepare_params()
+        response = requests.get(url, params, headers=self.headers)
+        self._check_response_status(response)
+        return response.json()
+
+    def get_cities_with_content_on_requested_languages(self, languages=None):
+        """
+        Get list of cities that have content on requested languages.
+        Args:
+            languages (str): Requested languages.
+        Returns:
+            list: List of cities.
+        """
+        url = "{base_url}cities".format(base_url=self.api_url)
+        if languages is None:
+            params = self._prepare_params()
+        else:
+            params = self._prepare_params(languages=languages)
+        response = requests.get(url, params, headers=self.headers)
+        self._check_response_status(response)
+
+        cities_data = response.json()
+        cities = []
+        if cities_data:
+            for city in cities_data:
+
+                city_title = city.get("title")
+                city_uuid = city.get("uuid")
+                cities.append({"city_title": city_title,
+                               "city_uuid": city_uuid})
+            return cities
+        return cities
+
+
+client = IziTravelClient()
```

Comment:
It would be great to have some unittests for better understanding hot this works.

It's not critical, but would be great.